rsmas_logging.py

```diff
@@ -0,0 +1,71 @@
+import logging
+
+class rsmas_logger():
+
+    format = "%(levelname)s - %(message)s"
+    console_handler, file_handler = None, None
+    logfile_name = None
+    logger = logging.getLogger()
+
+    def __init__(self, console=True, file=None):
+
+        self.logfile_name = file
+
+        self.logger.setLevel(logging.DEBUG)
+        self.set_format(self.format)
+
+
+    def setup_filehandler(self, formatter):
+        file_handler = logging.FileHandler(self.logfile_name, 'a+', encoding=None)
+        file_handler.setLevel(logging.INFO)
+        file_handler.setFormatter(formatter)
+        return file_handler
+
+    def setup_consolehandler(self, formatter):
+        console_handler = logging.StreamHandler()
+        console_handler.setLevel(logging.INFO)
+        console_handler.setFormatter(formatter)
+        return console_handler
+
+    def set_format(self, new_format):
+
+        if self.console_handler is not None:
```

Comment:
Is this check necessary if your `console_handler` and `file_handler`s are dynamic members instead of static members of your class?

Also, more idiomatically: 
``` 
if self.console_handler:
    ...
```

Reply:
The `set_format` method is intended to be used to allow for setting custom logging formats per file. Obviously, it sets up a default logger with a default format on object instantiation, but, if the user wants to change from the default format, we need to clear the handler objects and make new ones with the new format. The check stops any null/None references having methods called on them ie. `self.logger.removeHandler(self.console_handler)`.

`if self.console_handler is not None` CAN be replaced by `if self.console_handler:`.